docs/reference/bpmn-processes/business-rule-tasks/business-rule-tasks.md

```diff
@@ -0,0 +1,68 @@
+---
+id: business-rule-tasks
+title: "Business rule tasks"
+---
+
+A business rule task is a special kind of task. Usually, it is used to show that a task evaluates a
+business rule, for example, a decision modeled in [DMN](https://www.omg.org/dmn/) (Decision Model
+and Notation).
+
+![task](assets/business-rule-task.png)
+
+Business rule tasks behave **exactly** like [service tasks](../service-tasks/service-tasks.md). Both
+task types are based on jobs and [job workers](../../../product-manuals/concepts/job-workers.md).
+The differences between these task types are the visual representation (i.e. the task marker) and
+the semantics for the model.
+
+When a process instance enters a business rule task then it creates a corresponding job and wait
+for its completion. A job worker should pick up the job and process it. When the job is completed
+then the process instance continues.
+
+:::info 
+
+Business rule tasks are not processed by Zeebe itself. In order to process a business rule task, you
+need to provide a job worker.
```

Comment:
💭 Here I split up the sentences into lines. In my opinion markdown files are better supported by git if each sentence is a new line. Its not a part of the styleguide so just wanted to mention it

Reply:
I'm not sure about the formatting. I handled it like other source code and wrap the line when reaching 100 characters.

Maybe, we can set up a formatter similar to Java code.

Reply:
>Maybe, we can set up a formatter similar to Java code.

That would be great. I know that prettier has [markdown support](https://prettier.io/blog/2017/11/07/1.8.0.html#markdown-support), which also does line wrapping for text through [prose-wrap](https://prettier.io/docs/en/options.html#prose-wrap) and is a node package so should work with the development environment of this repo. 

Personally, I'm no fan of the line wrapping at fixed line length for Markdown, but at least it would be consistent.

I'll open an issue to continue this topic.